crm_deduplicate_acl/wizards/partner_merge.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import SUPERUSER_ID, api, models
+
+
+class BasePartnerMergeAutomaticWizard(models.TransientModel):
+    _inherit = "base.partner.merge.automatic.wizard"
+
+    @api.cr_uid_context
+    def _merge(self, cr, uid, partner_ids, dst_partner=None, context=None):
```

Comment:
I don't see the need to duplicate efforts, api takes care of morphing calls from v8.